frontend/app/components/personal-log/table.js

```diff
@@ -0,0 +1,44 @@
+import Component from "@glimmer/component";
+import { inject as service } from "@ember/service";
+import { action } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+
+export default class TableComponent extends Component {
+  @service store;
+  @service router;
+  @service intl;
+  @service notify;
+
+  @tracked logs = [];
+  loadAmount = 25;
+
+  constructor() {
+    super(...arguments);
+    this.getLogs();
+  }
+
+  @tracked
+  canLoadMore = true;
+
+  @action
+  getLogs() {
+    this.store
+      .query("paper-trail-version", {
+        load: this.loadAmount
+      })
+      .then((res) => {
+        this.logs = res;
+        this.toggleLoadMore();
+      });
+  }
+
+  @action
+  loadMore() {
+    this.loadAmount += 25;
```

Comment:
Sooo you are currently always loading all the logs instead of just loading the next 25.
This seems like a heavy performance problem to me :thinking: 
Please change it so it just loads the next 25 instead of always loading all the previously loaded ones + 25.

Reply:
Should be fixed